### PR TITLE
[GC] Add notes for June 16th meeting

### PR DESCRIPTION
**Pull request by binji**

### CODE REVIEW

**Comment by binji** [comment]
@jakobkummerow @skuzmich @fgmccabe please add links for your slides. Thanks!

File: gc/2020/GC-06-16.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+JK: I'm hoping in a week or so, but not willing to make
+
+KM: Any further work?
+
+JK: Not quite done with milestone 2, not sure what to do after that. Don't want to waste work. Not too willing to do experimental work. We're interested in hearing feedback. Planning to take time to respond to feedback.
+
+HH: How best to try this out?
+
+JK: Compiling from source is not too hard. You can use JSVU, you can get fairly up-to-date binary snapshots. Also can use Chrome canary (24-hour latency)
```
Comment (RossTate) [inline gc/2020/GC-06-16.md:81]:
```suggestion
JK: Compiling from source is not too hard. You can use [JSVU](https://github.com/GoogleChromeLabs/jsvu), you can get fairly up-to-date binary snapshots. Also can use Chrome canary (24-hour latency)
```
Adding useful link that Adam provided on the chat.

File: gc/2020/GC-06-16.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+* Ross Tate
+* Ryan Hunt
+* Sabine
+* Syvatoslav Kuzmich
+* Wouter van Oortmersson
+* Zalim
+
+### Update: v8 GC prototype (@jakobkummerow)
+
+[Jacob presenting]
```
Comment (jakobkummerow) [inline gc/2020/GC-06-16.md:58]:
```suggestion
[Jakob presenting]
```

File: gc/2020/GC-06-16.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+TODO(jkummerow): slides
```
Comment (jakobkummerow) [inline gc/2020/GC-06-16.md:59]:
```suggestion
[slides](https://docs.google.com/presentation/d/1FhDU3lkjQIuXA7K9MK3MXggQvxT5Glj4e2bifr4_Jlg/edit?usp=sharing)
```

File: gc/2020/GC-06-16.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+[some audio issues]
+
+RT: Function calls have to be done by putting a funcref into a table, just wondering if that's the plan
+
+JK: Yes, I think so. Not strictly required to have function references right now. If so, we'll add it.
+
+Zalim: Any information about when it will be finished?
+
+JK: I'm hoping in a week or so, but not willing to make
```
Comment (jakobkummerow) [inline gc/2020/GC-06-16.md:73]:
```suggestion
JK: I'm hoping in a week or so, but hesitant to make any promises.
```

**Comment by binji** [comment]
@skuzmich @fgmccabe friendly ping! Please provide links to your slides. I'll land for now and we can update later.

**Comment by skuzmich** [comment]
Kotlin/Wasm
https://docs.google.com/presentation/d/1shBSyexgdwegaGQH3PXtFiLcyWDsi4BvYmi6n0hVIj8/edit?usp=sharing

**Pull request merged**